src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/repositories/CIModuleRepository.java

```diff
@@ -0,0 +1,30 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.repositories;
+
+import com.novi.backend_spring_boot_tech_it_easy_controller.models.CIModule;
+import com.novi.backend_spring_boot_tech_it_easy_controller.models.Television;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+
+@Repository
+public interface CIModuleRepository extends JpaRepository<CIModule, Long> {
+//    List<Television> findByType(String type);
```

Comment:
Niet heel spannend maar zou evt bij pr je comment zoals deze niet meenemen in je git commit